paper/src/main/kotlin/ltd/redeye/vanguard/paper/network/PaperSingleMessagingProxy.kt

```diff
@@ -12,21 +13,21 @@ import java.util.*
  */
 object PaperSingleMessagingProxy: MessagingProxy {
 
-    override fun alertPlayer(uuid: UUID, message: VanguardMessage) {
+    override fun alertPlayer(uuid: UUID, message: VanguardMessage, placeholders: TagResolver?) {
         val player = Bukkit.getPlayer(uuid)
         if(player != null) {
```

Comment:
Could you run this through your editorconfig styler and re-commit?

Code content in itself is fine, and this needs to be automated in a separate issue.

Reply:
Re-ran @PhilCCMG - We should put an editorconfig and checkstyle onto this repository in the near future, I'll add it to issues

Reply:
Yeah, I've attached it via https://github.com/redeyetechnology/vanguard/pull/8